### PR TITLE
SONARAZDO-402 Make unit tests fail build CI

### PR DESCRIPTION
**Pull request by 7PH**
Failing unit tests were not failing the cirrus build job! (see [ticket](https://sonarsource.atlassian.net/browse/SONARAZDO-402))

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
     "format-check": "prettier --list-different \"src/**/*.{js,ts}\"",
     "lint": "eslint --ext js,ts --quiet \"src/**/*.{js,ts}\"",
     "lint-report": "eslint --ext js,ts -f json -o eslint-report.json \"src/**/*.{js,ts}\"",
-    "test": "find src/common -mindepth 1 -maxdepth 1 -type d -exec sh -c 'npm run test --prefix \"$0\"' {} \\;",
```
Comment (7PH) [inline package.json:17]: `find .. -exec` isn't forwarding the exit code

File: src/common/latest/helpers/__tests__/api-test.ts [hunk 2]
```diff
@@ -134,7 +134,7 @@ describe("fetchMetrics", () => {
     const metrics = await fetchMetrics(MOCKED_ENDPOINT);
     expect(get).toHaveBeenCalledWith(MOCKED_ENDPOINT, "/api/metrics/search", true, {
       f: "name",
-      ps: 500,
+      ps: "500",
```
Comment (7PH) [inline src/common/latest/helpers/__tests__/api-test.ts:137]: this was making a test fail

File: src/common/latest/jest.config.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
 const path = require("path");
 
+const esModules = ["node-fetch", "data-uri-to-buffer", "fetch-blob", "formdata-polyfill"].join("|");
```
Comment (7PH) [inline src/common/latest/jest.config.js:3]: this is needed to transform with babel the `node-fetch` dependency for jest to be able to run the tests without mocking the entire module

**Review by stanislavhh — APPROVED** [review]
nice work, lgtm 👍

**Pull request merged**